.gitignore

```diff
@@ -6,10 +6,9 @@
 .DS_Store
 *.geojson
 *.parquet
-*.csv
```

Comment:
I removed this global restriction since we do in fact want CSVs in the `dbt/seeds` directory to be stored in version control. We could also consider configuring an exception for the `dbt/seeds/` directory, but that seemed a little more complicated and I didn't feel like messing with it right now; instead I just ignored a specific expected CSV file in the `dbt/.gitignore` file below.

Reply:
Thanks for figuring that out, done in 406bf885a2f78007d4435c8f31623b8138ff389d!